hardware/ThermostatHardware.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <iostream>
+#include "DomoticzHardware.h"
+
+class CThermostatHardware : public CDomoticzHardwareBase
+{
+public:
+
+	//thermostat available mode in string "OFF,ECO,CONFOR,AUTO," separated by comma, 
+	// and in the order from 0 to N and 0 = OFF 1 = ECO, 2 = CONFOR 3 = AUTO
+	std::string  AvailableMode;
```

Comment:
```suggestion
	std::string AvailableMode;
```
Use a code formatter for new code